Sources/SlideKit/Views/PresentationView.swift

```diff
@@ -29,7 +29,9 @@ public struct PresentationView<Content>: View where Content: View {
 
     public var body: some View {
         ZStack(alignment: .center) {
+#if !os(visionOS)
             Color.black
+#endif
```

Comment:
To make the intention more clear and improve the readability, could you define this code as an extension of `Color` like this?

```swift
extension Color {
    /// A default color for the background of a slide.
    /// In visionOS, the color is `clear` to enable glass effects for the background.
    /// In other platforms, the color is `black`.
    public static var slideDefaultBackground: Color {
#if os(visionOS)
        Color.clear 
#else
        Color.black
#endif
    }
}
```

Reply:
That's good!